jwql/website/apps/jwql/bokeh_dashboard.py

```diff
@@ -37,7 +37,7 @@
 
 from bokeh.layouts import column
 from bokeh.models import Axis, ColumnDataSource, DatetimeTickFormatter, HoverTool, OpenURL, TapTool
-from bokeh.models import TabPanel, Tabs
+from bokeh.models.layouts import TabPanel, Tabs
```

Comment:
In bokeh_containers.py these imports do not include "layouts"

Reply:
TabPanel and Tabs now exists in layouts as of bokeh 3.0

Reply:
So then should the import statement in bokeh_containers.py be updated to include "layouts"?

Reply:
100% - Good catch